phpzlc/phpzlc/2.4/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "PHPZlc\\PHPZlc\\Bundle\\PHPZlcBundle": ["all"]
+    },
+    "copy-from-package": {
+        "Doctrine/ORM/Rewrite/Templates/Repository.tpl.php": "vendor/symfony/maker-bundle/src/Resources/skeleton/doctrine/Repository.tpl.php",
```

Comment:
What if Marker bundle is not installed?

Reply:
composer package ` phpzlc/phpzlc` will decide who to rely on